src/Components/LayoutWrapper/LayoutWrapper.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import HomeScreen from "../../screens/HomeScreen/HomeScreen";
+import HeaderPanel from "../HeaderPanel/HeaderPanel";
+
+import bemCssModule from "bem-css-modules";
+import { default as LayoutWrapperStyles } from "./LayoutWrapper.module.scss";
+const style = bemCssModule(LayoutWrapperStyles);
+
+const LayoutWrapper = () => {
+  return (
+    <div className={style()}>
+      <HeaderPanel />
+      <HomeScreen />
+    </div>
+  );
+};
```

Comment:
You should return children form params instead of <HeaderPanel /> and another components, because you will wrap every your screen with this wrapper. It should be only responsible for layout of each screen independently from its content.